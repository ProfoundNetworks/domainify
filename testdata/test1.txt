www.munichre.com
www.gatesofcapernum.a-zcompanies.com
en.wikipedia.org
askubuntu.com
www.oklahomaendurancelacrosse.teamsnapsites.com
