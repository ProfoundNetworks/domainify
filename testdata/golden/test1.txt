munichre.com
gatesofcapernum.a-zcompanies.com
wikipedia.org
askubuntu.com
oklahomaendurancelacrosse.teamsnapsites.com
